SConstruct

```diff
@@ -120,6 +120,7 @@ opts.Add(BoolVariable('deprecated', "Enable deprecated features", True))
 opts.Add(BoolVariable('gdscript', "Enable GDScript support", True))
 opts.Add(BoolVariable('minizip', "Enable ZIP archive support using minizip", True))
 opts.Add(BoolVariable('xaudio2', "Enable the XAudio2 audio driver", False))
+opts.Add(BoolVariable('vulkan_enabled', "Enable vulkan", True))
```

Comment:
Some related discussion here: https://github.com/godotengine/godot-proposals/issues/190#issuecomment-548043271, it's probably better to have something like one `renderer=list,of,renderer` flag instead of individual one for each.